javav2/example_code/rekognition/Readme.md

```diff
@@ -35,6 +35,18 @@ Define these values to successfully run the JUnit tests:
 - **celId** - The ID value of the celebrity. You can use the **RecognizeCelebrities** example to get the ID value.
 - **moutainImage** - The path to an image that contains mountains.
 - **collectionName** - A string value that represents the collection name.
+- **ppeImage** - An image that contains a person wearing a mask. 
```

Comment:
Are there any specific formats for the image? Gif, jpeg, ???.

Reply:
Fixed